src/status_im2/contexts/chat/messages/pin/banner/view.cljs

```diff
@@ -4,14 +4,12 @@
 
 (defn banner
   [chat-id]
-  (let [pinned-messages (rf/sub [:chats/pinned chat-id])
-        latest-pin-id   (-> pinned-messages
-                            vals
-                            first
-                            (get :message-id))
-        latest-pin-text (get-in (rf/sub [:chats/chat-messages chat-id])
```

Comment:
this second sub seems unnecessary as there is access to latest-pin-text from the first sub

Reply:
Ah 😑